catalog/README.md

```diff
@@ -0,0 +1,19 @@
+# Catalog
+
+This package holds i18n for DBaaS Controller.
+
+[gotext](https://pkg.go.dev/golang.org/x/text@v0.3.3/cmd/gotext?tab=doc)
+
+To translate, we need to follow the next steps.
+
+- Run `make gen` to collect all messages from source code; it also merges them with existing translation in `messages.gotext.json` and writes them into `out.gotext.json`.
```

Comment:
Shouldn't we call `gotext extract` for that?

Reply:
For this particular step `gotext extract` can be as well.  BTW `gotext update` also extract all messages plus generate code. So we can use `gotext update` for both steps.